mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping

```diff
@@ -13,7 +13,17 @@ CLASS dad net/minecraft/client/render/entity/model/BipedEntityModel
 	METHOD <init> (FFII)V
 		ARG 1 scale
 		ARG 3 textureWidth
+		ARG 4 textureHeight
+	METHOD a ()Ldcj;
+	METHOD a lerpAngle (FFF)F
+		ARG 1 a
+		ARG 2 b
+		ARG 3 t
+	METHOD a (FLaht;)V
```

Comment:
Missing mapping?

Reply:
There's a magic method to get rid of these artifacts (@gegy1000 please fix):

1. gradle yarn
2. save
3. close enigma

Reply:
This issue is hopefully resolved in https://github.com/FabricMC/Enigma/commit/b34305a235cdaaeadecad874931f2f2b62971711

Reply:
After resaving:
![screenshot from 2019-02-16 19-42-52](https://user-images.githubusercontent.com/8060034/52902553-5f8f9400-3223-11e9-9206-b3efcf83b163.png)

Reply:
No! Not manually!